provision.sh

```diff
@@ -27,6 +27,12 @@ wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
 echo "--------------- Installing both virtualenv and virtualenvwrapper ---"
 pip install virtualenvwrapper virtualenv
 
+# install postgresql and setup user
+echo "--------------- Installing postgresql ------------------------------"
+apt-get install -y python-dev libpq-dev postgresql postgresql-contrib
+su - postgres -c "createuser -s vagrant"
+su - vagrant -c "createdb $DB"
```

Comment:
any specific reason @alayek ? i followed [this](https://github.com/torchbox/vagrant-django-template/blob/master/etc/install/install.sh#L50)